src/M6Web/Component/Statsd/MessageEntity.php

```diff
@@ -118,13 +130,45 @@ public function getUnit()
     }
 
     /**
+     * @return array
+     */
+    public function getTags()
+    {
+        return $this->tags;
+    }
+
+    /**
+     * @return string Tags formatted for sending
+     * ex: "server=5,country=fr"
+     */
+    private function getTagsAsString()
+    {
+        $tags = array_map(function($k, $v) {
+            return $k.'='.$v;
+        }, array_keys($this->getTags()), $this->getTags());
```

Comment:
on peut se passer de l'overhead de getTags() en appelant directement $this->tags je pense